recipe/meta.yaml

```diff
@@ -25,7 +25,7 @@ requirements:
     - setuptools
   run:
     - python
-    - conda-build-all >=1.0.3
+    - conda-build-all >=1.1.0
```

Comment:
Should we just do 1.1.1? Otherwise we have incompatibilities with some `conda` versions.